client/src/components/App.jsx

```diff
@@ -22,6 +22,11 @@ const ImageView = styled.div`
   display: flex;
   margin: 0px;
   margin-top: 20px;
+  transition: ${(props) => props.sliding ? 'none' : 'transform 1s ease'};
+  transform: ${(props) => {
+    if (!props.sliding) return 'translateX(calc(-33.3333%))'
+    return 'translateX(0%)'
```

Comment:
I would stick to the ternary operator you used on line 25 for readability, 
transform: ${(props) => props.sliding? 'translateX(0%)' : 'translateX(calc(-33.3333%))'};